lib/ansible/plugins/inventory/netbox.py

```diff
@@ -312,9 +325,16 @@ def refresh_url(self):
         query_parameters.extend(filter(lambda x: x,
                                        map(self.validate_query_parameters, self.query_filters)))
         self.device_url = self.api_endpoint + "/api/dcim/devices/" + "?" + urlencode(query_parameters)
+        self.virtual_machines_url = "".join([self.api_endpoint,
```

Comment:
Just in case, this syntax could be used:
```
self.virtual_machines_url = (self.api_endpoint +
                             "/api/virtualization/virtual-machines/?" +
                             urlencode(query_parameters))
```

Reply:
What benefit would it bring?

Reply:
No change required here, I meant: parentheses could be used in order to split expression across lines (instead of calling `join`).